src/main/java/uk/nhs/digital/nhsconnect/nhais/model/fhir/ParameterNames.java

```diff
@@ -0,0 +1,13 @@
+package uk.nhs.digital.nhsconnect.nhais.model.fhir;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+@RequiredArgsConstructor @Getter
+public enum ParameterNames {
+    GP_TRADING_PARTNER_CODE("gpTradingPartnerCode"),
```

Comment:
@grzegorz-robak did that in his PR so we'll have to consolidate those changes into one place